lib/fog/kubevirt/compute/compute.rb

```diff
@@ -356,7 +357,7 @@ def create_client_from_token(url)
               :bearer_token => @kubevirt_token
             }
           }
-          version = detect_version(url.to_s, @opts[:ssl_options])
+          version = @kubevirt_version || detect_version(url.to_s, @opts[:ssl_options])
```

Comment:
same logic should be applied for `create_client_from_config` (line 379)

Reply:
done